backend/handlers/handlers.go

```diff
@@ -243,6 +243,7 @@ func NewHandlers(
 	getAPIRouterNoError(apiRouter)("/exchange/deals/{action}/{code}", handlers.getExchangeDeals).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/exchange/supported/{code}", handlers.getExchangeSupported).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/exchange/btcdirect-otc/supported/{code}", handlers.getExchangeBtcDirectOTCSupported).Methods("GET")
+	getAPIRouter(apiRouter)("/exchange/btcdirect-otc/info/{action}/{code}", handlers.getBtcDirectInfo).Methods("GET")
```

Comment:
just tiny nit: probably we should drop the `-otc` part for api endpoints that are for both the widget and the otc link.

In a past release we had only the OTC link so then it made sense.